src/rqt_publisher/publisher.py

```diff
@@ -353,6 +353,7 @@ def remove_publisher(self, publisher_id):
         publisher_info = self._publishers.get(publisher_id, None)
         if publisher_info is not None:
             publisher_info['timer'].stop()
+            self._node.destroy_publisher(publisher_info['publisher'])
```

Comment:
Is the same also missing in `clean_up_publishers`? I don't see where publishers are cleaned up when the plugin is shut down.